src/test/java/io/github/pavelapis/minesweeper/frames/WelcomeFrameTest.java

```diff
@@ -75,7 +75,7 @@ void testCorrectStartParameters() {
         columnInputField.setText("10");
         minesInputField.setText("20");
 
-        assertThat(welcomeFrame.isDisplayable()).isTrue();
+        assertThat(welcomeFrame != null && welcomeFrame.isDisplayable()).isTrue();
```

Comment:
```suggestion
        assertThat(welcomeFrame).isNotNull();
        assertThat(welcomeFrame.isDisplayable()).isTrue();
```

Reply:
Such code fails with spotbugs error: `Low: WelcomeFrameTest.welcomeFrame not initialized in constructor and dereferenced in io.github.pavelapis.minesweeper.frames.WelcomeFrameTest.testCorrectStartParameters() [io.github.pavelapis.minesweeper.frames.WelcomeFrameTest] At WelcomeFrameTest.java:[line 79] UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR`

Reply:
Just ignore `UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR`